src/azul/service/manifest_service.py

```diff
@@ -815,15 +823,44 @@ def manifest_config(self) -> ManifestConfig:
                               combine_script='return new ArrayList(params._agg.fields)',
                               reduce_script=reduce_script)
         es_search = es_search.extra(size=0)
-        response = es_search.execute()
-        assert len(response.hits) == 0
+        start = time.time()
+        try:
+            response = es_search.execute()
+        # If timeout is exceeded the request is partitioned to reduce load on ES
+        except ConnectionTimeout:
+            logger.warning('Elasticsearch request exceeded timeout limit, retrying …')
+            fields = self._partition_search(es_search)
```

Comment:
I figure we try to do a normal request to ES, without async requests, is that acceptable?

Reply:
I doubt that's a good idea. While partitioning the search adds load to ES for small manifests, trying an unpartitioned search first wastes a lot of time for large manifests and risks poisoning the lambda.